benchmarks/src/main/java/io/druid/benchmark/indexing/IncrementalIndexReadBenchmark.java

```diff
@@ -106,13 +106,18 @@ public void setup() throws IOException
 
   }
 
+  protected boolean isRollup() {
```

Comment:
instead of having this as a function, can we parameterize the benchmark similar to what we do for tests?